interchaintest/docker.go

```diff
@@ -29,7 +30,15 @@ type dockerErrorDetail struct {
 	Message string `json:"message"`
 }
 
-func BuildRelayerImage(t *testing.T) {
+func uniqueRelayerImageName() (string, error) {
+	uuid, err := uuid.NewRandom()
+	if err != nil {
+		fmt.Printf("Failed to generate UUID: %v\n", err)
```

Comment:
instead of fmt.Printf and returning the error, just `return "", fmt.Errorf("failed to generate UUID: %w", err)`